ngx_http_txid120_logic.c

```diff
@@ -16,7 +17,9 @@ void ngx_http_txid120_logic(FILE* urandom, uint8_t* txid120) {
   }
 
   // fill remaining 8 bytes with random data
-  fread(&txid[7], 8, 1, urandom);
+  if (fread(&txid[7], 8, NMEMB, urandom) != NMEMB) {
+    printf("failed reading /dev/urandom");
```

Comment:
This should at the least be using fprintf(STDERR, ...).  I'll discuss with @topaz whether he prefers:

- stderr
- nginx logging 
- changing the interface to return an error code and allowing the consumer to deal with it
- masking the error by zero-padding the txid